cfgov/unprocessed/apps/teachers-digital-platform/__mocks__/copy-to-clipboard.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
This looks like you could have `/* eslint-disable-next-line no-undef */` instead, so that it only covers the `jest`, not the whole file.

Reply:
True! The whole file is just the following though:

```
const spy = jest.fn();

module.exports = spy;
```

Reply:
Fixed in 2e529be4eb50be9c927b4aeb046a8eee6bddb9df